packages/atomic/src/autoloader/index.ts

```diff
@@ -1,5 +1,5 @@
 //@ts-expect-error TODO: Simplify path to target some kind of index file?
-import searchElementMap from '../components/components/search/lazy-index.js';
+import searchElementMap from '../components/components/lazy-index.js';
```

Comment:
Maybe we can rename the import since it's no longer only the search elements

could be simply `elementMap`